src/protocol/attribution/input.rs

```diff
@@ -19,9 +19,52 @@ use std::marker::PhantomData;
 use typenum::Unsigned;
 
 //
-// `accumulate_credit` protocol
+// 'apply_attribution_window` protocol
 //
 #[derive(Debug)]
+pub struct ApplyAttributionWindowInputRow<F: Field, BK: Fp2Array> {
+    pub timestamp: AdditiveShare<F>,
+    pub is_trigger_report: AdditiveShare<F>,
+    pub helper_bit: AdditiveShare<F>,
+    pub breakdown_key: XorShare<BK>,
+    pub trigger_value: AdditiveShare<F>,
+}
+
+#[derive(Debug)]
+pub struct MCApplyAttributionWindowInputRow<F: Field, T: Arithmetic<F>> {
+    pub timestamp: T,
```

Comment:
I do not like this name. This actually indicates a _time delta_, not a timestamp. That confused me when I was looking at the test and expected to see monotonically increasing timestamps.

Reply:
nevermind... it actually is timestamps =)